### DISCUSSION TITLE
Loading zfsbootmenu via PXE

### DISCUSSION THREAD

**Discussion by mabra** [zbm-dev/zfsbootmenu#630] [Feature Requests]
This would make zfsbootmenu the most universal boot/repair method for ZFS setups!

**Comment by zdykstra** [comment]
This is already doable with the EFI build (loaded via iPXE) or with the component builds (loaded via pxelinux over TFTP).

**Reply by mabra** [reply]
Thanks!
Will take a look later onto this!

**Reply by juju4** [reply]
Tried PXE with grub and ipxe but not working. Any config example?
Thanks

```
# grub menu
# FIXME! "error: invalid magic number"
menuentry "ZFSBootMeny 2.3.0" {
   linux /zfsbootmenu-release-x86_64-v2.3.0-vmlinuz.EFI
}

# ipxe menu
set zfsbootmenu-kernel ${boot-url}zfsbootmenu-release-x86_64-v2.3.0-vmlinuz.EFI
menu ZFSBootMenu
item zfsbootmenu ${space} ZFSBootMenu
kernel ${zfsbootmenu-kernel}

```
